src/main/java/org/reactome/release/cosmicupdate/COSMICUpdateUtil.java

```diff
@@ -0,0 +1,320 @@
+package org.reactome.release.cosmicupdate;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.sql.SQLException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVPrinter;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.schema.InvalidAttributeException;
+
+/*
+ * Thie class contains utility methods that are to be used for updating COSMIC identifiers.
+ */
+public class COSMICUpdateUtil
+{
+	static final String COSMIC_LEGACY_PREFIX = "COSM";
+	private static final String COSMIC_FUSION_ID = "FUSION_ID";
+	private static final String COSMIC_GENOMIC_MUTATION_ID = "GENOMIC_MUTATION_ID";
+	private static final String COSMIC_MUTATION_ID = "MUTATION_ID";
+	private static final String COSMIC_LEGACY_MUTATION_ID = "LEGACY_MUTATION_ID";
+	static final String COSMIC_FUSION_PREFIX = "COSF";
+	private static final Logger logger = LogManager.getLogger();
+	private static String dateSuffix;
+	private static String reportsDirectoryPath = "reports";
```

Comment:
Maybe have this as type `Path` here so there is no need to convert it later on?

Reply:
It is used as a prefix to to a path to a file, so a String is more appropriate.